src/main/scala/cromiam/cromwell/CromwellClient.scala

```diff
@@ -53,5 +71,13 @@ object CromwellClient {
   // See: https://github.com/akka/akka-http/issues/64
   val TimeoutAccessHeader = "Timeout-Access"
 
-  private case class CromwellConnectionFailure(f: Throwable) extends Exception(s"Unable to connect to Cromwell (${f.getMessage})", f)
+  final case class CromwellConnectionFailure(f: Throwable) extends Exception(s"Unable to connect to Cromwell (${f.getMessage})", f)
+
+  implicit class EnhancedWorkflowLabels(val wl: WorkflowLabels) extends AnyVal {
+    import Collection.CollectionLabelName
+    import Collection.collectionJsonReader
+    def toCollection: Option[Collection] = {
```

Comment:
Could this be named `caasCollection` or similar, perhaps with a comment? 

`to` makes it seem like the whole instance is being used in some new capacity, however it is a subset of the `workflowlabel`'s fields.

and `caas` because it isn't clear what this is a collection of.